<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panel_sim_storm</class>
 <widget class="QWidget" name="panel_sim_storm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>长度(单位：字节)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lenLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>次数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sendNumLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>发送间隔(单位ms)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="intervalLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="randRadioButton">
       <property name="text">
        <string>使用随机值</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="hexRradioButton">
       <property name="text">
        <string>按16进制发送（使用空格隔开如：0E 34 45）</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="commonRradioButton">
       <property name="text">
        <string>发送消息内容：</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTextEdit" name="sendInfoTextEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startPushButton">
       <property name="text">
        <string>启动(&amp;D)</string>
       </property>
       <property name="icon">
        <iconset resource="sim_iec61850.qrc">
         <normaloff>:/sim_iec61850/Resources/KEY04.ICO</normaloff>:/sim_iec61850/Resources/KEY04.ICO</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopPushButton">
       <property name="text">
        <string>停止(&amp;T)</string>
       </property>
       <property name="icon">
        <iconset resource="sim_iec61850.qrc">
         <normaloff>:/sim_iec61850/Resources/del.gif</normaloff>:/sim_iec61850/Resources/del.gif</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>159</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="sim_iec61850.qrc"/>
 </resources>
 <connections/>
</ui>
